pkg/run/run.go

```diff
@@ -75,31 +76,39 @@ func NewCmdRun() *cobra.Command {
 			// Fire off sonobuoy
 			err := o.Run(kclient, sclient)
 			if err != nil {
-				log.WithError(err).Error("Error running the tool. Please check the errors and try again.")
-				return
+				log.WithError(err).Fatal("Error running the tool. Please check the errors and try again.")
 			}
 
 			log.Info("Jobs scheduled! Waiting for resources be created...")
 
 			// Wait for Sonobuoy to create
 			wait.WaitForRequiredResources(kclient)
 			if err != nil {
-				log.WithError(err).Error("error waiting for sonobuoy pods to become ready")
-				return
+				log.WithError(err).Fatal("error waiting for sonobuoy pods to become ready")
 			}
 
+			// Sleep to give status time to appear
+			time.Sleep(status.StatusInterval)
+
 			s := status.NewStatusOptions(o.watch)
 			err = s.WaitForStatusReport(cmd.Context(), sclient)
 			if err != nil {
-				log.WithError(err).Error("error retrieving aggregator status")
+				log.WithError(err).Fatal("error retrieving aggregator status")
 			}
 
-			// TODO Why's there a second StatusOptions instance?
-			st := status.NewStatusOptions(o.watch)
-			st.Update(sclient)
-			st.Print(cmd, sclient)
+			err = s.Update(sclient)
+			if err != nil {
+				log.Fatal(err)
```

Comment:
should a fatal error result in exiting this function?

Reply:
Yeah, that's the intent where the `log.Fatal` were added in this PR